content/03.combining-datasets.md

```diff
@@ -21,4 +21,4 @@ Ideally, the structure of the composite dataset is reflects differences in varia
 If the samples from the same cohort tend to group together regardless of phenotype, this suggests that the datasets used to generate the composite dataset need to be corrected to overcome differences in how the data were generated or collected.
 In the next section, we will discuss approaches that can aid in identifying and visualizing structure in datasets to determine whether composite rare disease datasets are appropriate for use in ML.
 
-![Combining datasets to increase training data TODO: expand this legend, B needs to cover training/testing, etc.](content/images/figures/pdfs/figure-1-combining-datasets.png){#fig:1}
\ No newline at end of file
+![Combining datasets to increase data for training machine learning models. a) Appropriate methods are required to combine smaller datasets into a composite dataset without loss of information: The left panel shows multiple small rare disease datasets that need to be combined to form a dataset of higher sample size. The color of the samples suggest class or groups present in the datasets. The shape correlates to the identity of the datasets. The middle panel shows methods that may be used to combine the datasets. The right panel shows principle component analysis of the combined datasets to verify proper integration without loss of information. b) Appropriately prepared composite datasets can be used to make training, evaluation, and validation datasets for machine learning: Left panel shows the division of the composite dataset into training dataset and a held-out validation dataset (top). The training set is further divided into training and evaluation datasets for k-fold cross-validation. The panel on the right shows the class composition of the training, evaluation, and held-out validation datasets](content/images/figures/pdfs/figure-1-combining-datasets.png){#fig:1}
```

Comment:
```suggestion
![Combining datasets to increase data for training machine learning models. a) Appropriate methods are required to combine smaller datasets into a composite dataset without loss of information: The left panel shows multiple small rare disease datasets that need to be combined to form a dataset of higher sample size. The color of the samples suggest classes or groups present in the datasets. The shape represents the dataset of origin. The middle panel shows methods that may be used to combine the datasets. The right panel shows Principal Component Analysis of the combined datasets to verify proper integration without loss of information. b) Appropriately prepared composite datasets can be used to make training, evaluation, and validation datasets for machine learning: Left panel shows the division of the composite dataset into training dataset and a held-out validation dataset (top). The training set is further divided into training and evaluation datasets for 4-fold cross-validation. The panel on the right shows the class distribution of the training, evaluation, and held-out validation datasets](content/images/figures/pdfs/figure-1-combining-datasets.png){#fig:1}
```

Reply:
I would say, more generally, I'm not sure we're concerned about _loss of information_ in the "without reprocessing" example as much as we're concerned about technical artifacts. You would want to make sure you didn't "overcorrect" after batch correction (e.g., you removed biological variation), but that's not what this figure is about in my opinion.

Reply:
I may be missing something, but I would consider 
> You would want to make sure you didn't "overcorrect" after batch correction (e.g., you removed biological variation)

as "loss of information".  :)

Reply:
also cant believe I wrote "principle component analysis" *facepalm*

Reply:
I agree with you about the loss of information being important. My concern is that the figure does not show that in my opinion.

Reply:
@jaclyn-taroni I modified the fig legend a bit more. Take a look when you have a chance and let me know what you think.